qemu/tests/migration_multi_host_with_kdump.py

```diff
@@ -0,0 +1,185 @@
+                                                    "1500"))
+            self.vmaddr = None
+            self.crash_timeout = float(params.get("crash_timeout", 360))
+            self.def_kernel_param_cmd = ("grubby --update-kernel=`grubby"
+                                         " --default-kernel`"
+                                         " --args=crashkernel=128M@16M")
+            self.kernel_param_cmd = params.get("kernel_param_cmd",
+                                               self.def_kernel_param_cmd)
+            def_kdump_enable_cmd = ("chkconfig kdump on &&"
+                                    " service kdump restart")
+            self.kdump_enable_cmd = params.get("kdump_enable_cmd",
+                                               def_kdump_enable_cmd)
+            def_crash_kernel_prob_cmd = ("grep -q 1 /sys/kernel/"
+                                         "kexec_crash_loaded")
+            self.crash_kernel_prob_cmd = params.get("crash_kernel_prob_cmd",
+                                                    def_crash_kernel_prob_cmd)
+            self.crash_cmd = params.get("crash_cmd",
+                                        "echo c > /proc/sysrq-trigger")
+            self.vmcore_chk_cmd = params.get("vmcore_chk_cmd",
+                                             "ls -R /var/crash | grep vmcore")
+            self.login_timeout = 360
+            self.nvcpu = int(params.get("smp", 1))
+            self.id = {"src": self.srchost,
+                       "dst": self.dsthost,
+                       "type": "file_transfer"}
+
+        def _prepare_vm(self, vm_name):
+
+            """
+            Prepare, start vm and return vm.
+
+            :param vm_name: Class with data necessary for migration.
+
+            :return: Started VM.
+            """
+            new_params = self.params.copy()
+            new_params['migration_mode'] = None
+            new_params['start_vm'] = 'yes'
+            self.vm_lock.acquire()
+            env_process.process(self.test, new_params, self.env,
+                                env_process.preprocess_image,
+                                env_process.preprocess_vm)
+            self.vm_lock.release()
+            vm = self.env.get_vm(vm_name)
+            vm.wait_for_login(timeout=self.login_timeout)
+            return vm
+
+        def ping_pong_migrate(self, sync, start_work, check_work):
+
+            for _ in range(self.migrate_count):
+                logging.info("ping pong migration...")
+                sync.sync(True, timeout=self.migration_timeout)
+                self.migrate_wait([self.vm], self.srchost, self.dsthost,
+                                  start_work=start_work,
+                                  check_work=check_work)
+                if self.migrate_count == 0:
+                    break
+                tmp = self.dsthost
+                self.dsthost = self.srchost
+                self.srchost = tmp
```

Comment:
python support swap two value like: self.dsthost, self.srchost = self.srchost, self.dsthost. below is my test.

>>> a = [1]
>>> b = [2]
>>> a,b = b, a 
>>> a
[2]
>>> b
[1]